### PR TITLE
Support Python 3.6

### PR DESCRIPTION
**Pull request by giginet**
```sh
$ pyenv install 3.6
$ pyenv local 3.6
$ python -m venv .venv
$ source .venv/bin/activate
```

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.6.0
```
Comment (giginet) [inline .python-version:1]: I think we prefer to add `.python-version` into repo.

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ sudo: false
 language: python
 python:
 - '3.5'
```
Comment (giginet) [inline .travis.yml]: We'll drop 3.5 support after migrating on production environment.

File: src/kawaz/core/templatetags/tests/test_shortenurl.py [hunk 3]
```diff
@@ -23,4 +23,4 @@ def test_shortenurl_templatetag(self):
 
         rendered = t.render(Context())
 
-        self.assertRegex(rendered, r'http:\/\/goo.\gl\/.+')
+        self.assertRegex(rendered, r'http:\/\/goo\.gl\/.+')
```
Comment (giginet) [inline src/kawaz/core/templatetags/tests/test_shortenurl.py:26]: 😱

**Comment by giginet** [comment]
@lambdalisue Please review 🙏

**Comment by lambdalisue** [comment]
LGTM. この量のテストが通るの爽快だよなー

**Pull request merged**